### ISSUE TITLE
No roaming calls can be made nor received

### ISSUE THREAD

**Issue by sm131313** [GrapheneOS/os-issue-tracker#2184]
Original topic where multiple user experience same.
https://discuss.grapheneos.org/d/4141-no-roaming-calls/ 

The other affected users are not in my country nor travelled to same country. 

I am currently on travel and experience issue with calling. No calls can be made and no calls can be received. Only mobile data works. The same thing happened to me on my last travel to, to another country where calling never worked. This is of course a critical issue as you are left overseas with no possibility to make any calls or receive calls. 

I have set roaming on. 
wifi calling on, with settings call through wifi, setting for roaming is set to wifi also.
4g calls on
Recommended network type to 5g (recommended)
Running main profile

I have attempted changing different settings but no luck. I then found the topic on forum where other users are reporting same issue with no fix. 
**Pixel 7
Version: TQ2A.230505.002.2023050500**

**Steps to reproduce:**
Be connected to a cellular network through roaming
Attempt to make a phonecall. It never connects the call, only attempts and then hangs up. 
If someone tries to call me, no signal comes on my phone either. Only an SMS is received once the person hangs up with "xxx tried to call you" this sms is a standard sms feature from my provider.

**Comment by thestinger** [comment]
No indication of any GrapheneOS specific issue, particularly since some people had LTE-only mode enabled.

**Comment by sm131313** [comment]
I have attempted all possible settings
3G, 4G, 4G Only, 5G as preferred network type
I have attempted enable and disable 4G-Calling in combination with above settings. The behaviour is same. 

I can see past tickets also: https://github.com/GrapheneOS/os-issue-tracker/issues/2101

Seems it has been reported multiple times by various users and no-one confirming finding any fix or work-around for it. 
Also nobody on Roaming has ever confirmed on any topic having calling to be working on their phones.
